lib/matplotlib/backends/_backend_tk.py

```diff
@@ -533,11 +533,26 @@ def set_cursor(self, cursor):
         window.configure(cursor=cursord[cursor])
         window.update_idletasks()
 
-    def _Button(self, text, file, command, extension='.gif'):
-        img_file = str(cbook._get_data_path('images', file + extension))
-        im = tk.PhotoImage(master=self, file=img_file)
-        b = tk.Button(
-            master=self, text=text, padx=2, pady=2, image=im, command=command)
+    def _Button(self, text, image_file, toggle, command):
+        if image_file is not None:
+            im = tk.PhotoImage(master=self, file=image_file)
+        else:
+            im = None
+        if not toggle:
+            b = tk.Button(master=self, text=text, padx=2, pady=2, image=im,
```

Comment:
Note that `padx/y` do not have any effect on images with buttons:
https://www.tcl.tk/man/tcl8.6/TkCmd/options.htm#M-padx
> Most widgets only use this option for padding text: if they are displaying a bitmap or image, then they usually ignore padding options.

These options could be removed.